zeq/hbp/vocabulary.cpp

```diff
@@ -60,6 +64,52 @@ zeq::Event serializeCamera( const std::vector< float >& matrix )
     return event;
 }
 
+::zeq::Event serializeimageRawRGBA( const imageRawRGBA& image )
+{
+    ::zeq::Event event( EVENT_IMAGE_RAW );
+    flatbuffers::FlatBufferBuilder& fbb = event.getFBB();
+    auto imageData = fbb.CreateVector( image.data, image.resX * image.resY * RGBA_NBYTES );
+
+    ImageRawRGBABuilder builder( fbb );
+    builder.add_resX( image.resX );
+    builder.add_resY( image.resY );
+    builder.add_data( imageData );
+
+    fbb.Finish( builder.Finish() );
+    return event;
+}
+
+imageRawRGBA deserializeimageRawRGBA( const ::zeq::Event& event )
+{
+    auto data = GetImageRawRGBA( event.getData( ));
+    imageRawRGBA result;
+    result.resX = data->resX();
+    result.resY = data->resY();
+
+    result.data = const_cast<uint8_t*>(data->data()->Data());
+
+    return result;
+}
+
+::zeq::Event serializeRequest( const lunchbox::uint128_t& eventType)
+{
+    ::zeq::Event event( EVENT_REQUEST );
+    flatbuffers::FlatBufferBuilder& fbb = event.getFBB();
+
+    RequestBuilder builder( fbb );
+    builder.add_eventLow( eventType.low() );
+    builder.add_eventHigh( eventType.high() );
+
+    fbb.Finish( builder.Finish( ));
+    return event;
+}
+
+lunchbox::uint128_t deserializeRequest( const ::zeq::Event& event )
+{
+    auto data = GetRequest( event.getData( ) );
+    return lunchbox::uint128_t( data->eventHigh(), data->eventLow() );
```

Comment:
Please unify your style. I see different spaces between/after braces. Line 115 for instance is the 'correct' one.